lib/block-supports/layout.php

```diff
@@ -145,6 +145,40 @@ function gutenberg_get_layout_style( $selector, $layout, $has_block_gap_support
 	return $style;
 }
 
+/**
+ * Generates the utility classnames for the given blocks layout attributes.
+ * This method was primarily added to reintroduce classnames that were removed
+ * in the 5.9 release (https://github.com/WordPress/gutenberg/issues/38719), rather
+ * than providing an extensive list of all possible layout classes. The plan is to
+ * have the style engine generate a more extensive list of utility classnames which
+ * will then replace this method.
+ *
+ * @param array $block_attributes Array of block attributes.
+ *
+ * @return array Array of CSS classname strings.
+ */
+function gutenberg_get_layout_classes( $block_attributes ) {
+	$class_names = array();
+
+	if ( empty( $block_attributes['layout'] ) ) {
+		return $class_names;
+	}
+
+	if ( ! empty( $block_attributes['layout']['orientation'] ) ) {
+		$class_names[] = 'is-' . sanitize_title( $block_attributes['layout']['orientation'] );
+	}
+
+	if ( ! empty( $block_attributes['layout']['justifyContent'] ) ) {
+		$class_names[] = 'is-content-justification-' . sanitize_title(  $block_attributes['layout']['justifyContent'] );
```

Comment:
Tiny linting issue 😀

```suggestion
		$class_names[] = 'is-content-justification-' . sanitize_title( $block_attributes['layout']['justifyContent'] );
```